mobile/src/main/java/com/truckmuncher/app/vendor/VendorHomeActivity.java

```diff
@@ -178,6 +207,41 @@ public void onLoaderReset(Loader<Cursor> loader) {
         // no-op
     }
 
+    private void showMenu() {
+        getSupportFragmentManager().beginTransaction()
+                .add(android.R.id.content, MenuAdminFragment.newInstance(selectedTruck.id), MenuAdminFragment.TAG)
+                .addToBackStack(null)
+                .setTransition(FragmentTransaction.TRANSIT_FRAGMENT_FADE)
+                .commit();
+    }
+
+    private void showWarning(int numItems) {
+        View checkBoxView = View.inflate(this, R.layout.dialog_items_unavailable_warning, null);
+        final CheckBox checkBox = (CheckBox) checkBoxView.findViewById(R.id.checkbox_dont_show_again);
+        DialogInterface.OnClickListener listener = new DialogInterface.OnClickListener() {
+            public void onClick(DialogInterface dialog, int id) {
+                if (checkBox.isChecked()) {
+                    SharedPreferences.Editor editor = sharedPreferences.edit();
+                    editor.putBoolean(getString(R.string.setting_item_unavailable_warning), false);
+                    editor.apply();
+                }
+
+                if (id == AlertDialog.BUTTON_POSITIVE) {
+                    showMenu();
+                }
+                dialog.cancel();
+            }
+        };
+
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setTitle(getString(R.string.title_items_unavailable));
+        builder.setMessage(String.format(getResources().getQuantityString(R.plurals.items_unavailable_message, numItems, numItems)))
```

Comment:
The first `numItems` parameter is not included in the format args, it's just used to figure out which plural message to use.